### PR TITLE
{tools}[GCCcore/11.3.0] xdotool v3.20211022.1, xclip v0.13

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
himem04.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/5418497396be477d1d6b8f4318ca5f4a for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/x/xdotool/xdotool-3.20211022.1-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+skipsteps = ['configure']
+
+installopts = 'PREFIX=%(installdir)s'
+
+builddependencies = [('binutils', '2.38')]
+
+dependencies = [('X11', '20220504')]
+
+sanity_check_paths = {
+    'files': ['bin/xdotool', 'include/xdo.h', 'lib/libxdo.so.3'],
```
Comment (jfgrimm) [inline easybuild/easyconfigs/x/xdotool/xdotool-3.20211022.1-GCCcore-11.3.0.eb:26]:
```suggestion
    'files': ['bin/xdotool', 'include/xdo.h', 'lib/libxdo.%s' % SHLIB_EXT],
```

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/5349e3ec9e766b8fe8bead8eb6e66637 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**